src/backend/functions/work-packages-create.ts

```diff
@@ -0,0 +1,125 @@
+
+import middy from '@middy/core';
+import jsonBodyParser from '@middy/http-json-body-parser';
+import httpErrorHandler from '@middy/http-error-handler';
+import validator from '@middy/validator';
+import { Handler } from 'aws-lambda';
+import { PrismaClient, WBS_Element_Status } from '@prisma/client';
+import {
+  buildSuccessResponse,
+} from 'utils';
+
+const prisma = new PrismaClient();
+
+const createWorkPackage: Handler = async ({ body }, _context) => {
+  
+    // get the corresponding project so we can find the next wbs number
+    // and what number work package this should be
+    const project = await prisma.project.findUnique({
+      where: {
+        projectId: body.projectId
+      },
+      include: {
+        goals: true,
+        features: true,
+        otherConstraints: true,
+        workPackages: { include: { wbsElement: true, dependencies: true } }
+      }
+    });
+  
+    if (project === null) throw new TypeError('Project Id not found!');
+  
+    const {workPackages} = project;
+    const {length} = workPackages;
+    const orderInProject = length;
+    // eslint-disable-next-line prefer-destructuring
+    const last = workPackages[length - 1];
+    const { wbsElement } = last;
```

Comment:
can we inline these? place them where they get used instead of having separate variables?